tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "sourceMap": true,                        /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
     "outDir": "./build",                      /* Redirect output structure to the directory. */
-    "rootDir": "./src",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
+    // "rootDir": "./src",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
```

Comment:
this is odd to me that we build out a src directory, but I'm seeing that that is a common directory to have in a final product of a react native solution

Reply:
I had to comment that option because now we are importing a module from `node_modules/mqtt/` and ts throws an error when trying to load a file from outside of `./src` anyway the folder is referenced in the `include` prop.